package.json

```
@@ -202,6 +202,6 @@
     },
     "engines": {
         "node": ">=16",
-        "pnpm": "8.7.4"
+        "pnpm": ">=8"
```

This will allow people to commit lockfiles changed with pnpm v9 which can cause unexpected issues.

Better if we can set a range.

```suggestion
        "pnpm": ">=8.7.4 <9"
```